### PR TITLE
Add extra tag to metrics deliverables YAML file.

### PR DESCRIPTION
**Pull request by ivadym**
## Description

We need to have unique tags in HK. `qc-metrics` tag will start to be used also for multiqc data (https://github.com/Clinical-Genomics/hermes/pull/95).

This PR adds this extra tag.

### Added
- `deliveable` tag to `<case_id>_metrics_deliverables.yaml`

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/constants/housekeeper_tags.py [hunk 1]
```diff
@@ -76,9 +76,9 @@ class HkMipAnalysisTag:
     SAMPLE_INFO: list[str] = ["sample-info"]
 
 
-class BalsamicAnalysisTag(StrEnum):
-    CONFIG: str = "balsamic-config"
-    QC_METRICS: str = "qc-metrics"
+class BalsamicAnalysisTag:
```
Comment (diitaz93) [inline cg/constants/housekeeper_tags.py:79]: it could be an `Enum` class as well

Comment (diitaz93) [reply cg/constants/housekeeper_tags.py:79]:
```suggestion
class BalsamicAnalysisTags:
```

**Review by diitaz93 — APPROVED** [review]
Looks great! Feel free to resolve my comments if not relevant

File: cg/constants/housekeeper_tags.py (hunk 1 above)
Comment (ivadym) [reply cg/constants/housekeeper_tags.py:79]: I'll keep it as is to align with the naming of the other pipeline analysis tags. Thank you for the review!

**Pull request merged**